src/packages/solid/components/ProgressBar/ProgressBar.tsx

```diff
@@ -48,18 +48,29 @@ const ProgressBar: Component<ProgressBarProps> = (props: ProgressBarProps) => {
   return (
     <View
       {...props}
-      style={props?.style?.Container ?? styles.Container}
+      style={[
+        ...[props.style].flat(),
```

Comment:
lets update the style prop to have the type of Partial<ProgressBarStyles> from the style file, and get rid of the ProgressBarStyleProps interface